math256/src/main.rs

```diff
@@ -0,0 +1,66 @@
+//! ETH compatible u256 math.
+
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#![warn(rust_2018_idioms)]
+#![deny(
+    dead_code,
+    nonstandard_style,
+    unused_imports,
+    unused_mut,
+    unused_variables,
+    unused_unsafe,
+    unreachable_patterns
+)]
+
+mod errors;
+mod math;
+mod result;
+#[cfg(test)]
+mod tests;
+
+use result::MathResult;
+
+use fluence::module_manifest;
+
+module_manifest!();
+
+pub fn main() {}
+
+use fluence::marine;
+
+/// adds 2 256 bits integers (ETH compatible)
```

Comment:
I think that it's better either to delete this comment or to comment in this way all other arithmetic functions in this module.

Reply:
will make comment one each